content/administration/install/cdn.rst

```diff
@@ -4,77 +4,98 @@ Setting up a Content Delivery Network (CDN)
+   custom URL.
 
-.. image:: cdn/keycdn_enable_CORS.png
-   :class: img-fluid
+.. image:: cdn/keycdn-zone.png
+   :align: center
+   :alt: KeyCDN's Zone configuration page.
 
-Once done, you'll have to wait a bit while KeyCDN_ is crawling your website.
+Under :guilabel:`General Settings` click the :guilabel:`Show all settings` button to expand the
+Zone options. This should be the last option on the page. After expanding the
+:guilabel:`General Settings` ensure that the :guilabel:`CORS` option is enabled. By default this
+option should be *enabled*.
 
-.. image:: cdn/keycdn_progressbar.png
-   :class: img-fluid
+Next, scroll to the bottom of the Zone configuration page and :guilabel:`Save` the changes. KeyCDN
+will indicate that the new Zone will be deployed. This can take about 10 minutes.
 
-.. note:: a new URL has been generated for your Zone, in this case it is
-          ``http://pulltest-b49.kxcdn.com``
+.. image:: cdn/zone-url.png
+   :align: center
+   :alt: KeyCDN deploying the new Zone.
 
-Step 2: Configure the odoo instance with your zone
---------------------------------------------------
+.. note::
+   A new :guilabel:`Zone URL` has been generated for your Zone, in this example it is
+   ``pulltest-xxxxx.kxcdn.com``. This will differ for each database.
+
+Copy this :guilabel:`Zone URL` to a Notepad as it will be used in the next steps.
 
-In the Odoo back end, go to the :guilabel:`Website Settings`: menu, then
-activate the CDN support and copy/paste your zone URL in the
-:guilabel:`CDN Base URL` field. This field is only visible and configurable if
-you have developer mode activated.
+Step 2: Configure the Odoo instance with the new zone
+-----------------------------------------------------
 
-.. image:: cdn/odoo_cdn_base_url.png
-   :class: img-fluid
+In the Odoo Website App, go to the :menuselection:`Settings` and then activate the
+:guilabel:`Content Delivery Network (CDN)` and copy/paste the :guilabel:`zone URL` in the
+:guilabel:`CDN Base URL` field. This field is only visible and configurable when
+:doc:`../../applications/general/developer_mode` is activated.
 
-Now your website is using the CDN for the resources matching the
-:guilabel:`CDN filters` regular expressions.
+.. note::
+   Ensure that there are two "//"" *Forward Slashes* before the :guilabel:`CDN Base URL` and one "/"
+   *Forward Slash* after the :guilabel:`CDN Base URL`.
 
-You can have a look to the HTML of your website in order to check if the CDN
-integration is properly working.
+:guilabel:`Save` the Settings.
 
-.. image:: cdn/odoo_check_your_html.png
-   :class: img-fluid
+.. image:: cdn/cdn-base-url.png
+   :align: center
+   :alt: Activate the CDN setting in Odoo.
 
+Now the website is using the CDN for the resources matching the :guilabel:`CDN filters` regular
+expressions.
 
-Why should I activate CORS?
----------------------------
+In the HTML of the Odoo website the CDN integration is evidenced as working properly by checking the
```

Comment:
Needs comma and use `abbr` tag for acronyms.
```suggestion
In the HTML of the Odoo website, the :abbr:`CDN (content delivery network)` integration is evidenced as working properly by checking the
```

Reply:
Thanks!